### ISSUE TITLE
check_ups shows OK when the UPS is ON ALARM

### ISSUE THREAD

**Issue by monssamonssa** [monitoring-plugins/monitoring-plugins#1708]
I'm writing because I found a behavior I want to know if it's correct.

Some days ago, we had one UPS overheat (fan failed) and we didn't get any notification, the plugin showed status "OK", we only found out because someone entered the server room and heard the beeping.

Checking the output of "upsc" we found this:
(fields I think relevant marked with asterisks)

```
battery.charge: 74
battery.voltage: 79.56
battery.voltage.high: 83.91
battery.voltage.low: 67.13
battery.voltage.nominal: 77.5
device.mfr: UPS
device.model: Model 3000
device.type: ups
driver.name: blazer_ser
driver.parameter.pollinterval: 2
driver.parameter.port: /dev/ttyS0
driver.parameter.synchronous: no
driver.version: 2.7.4
driver.version.internal: 1.57
input.current.nominal: 14.0
input.frequency: 50.0
input.frequency.nominal: 50
input.voltage: 212.0
input.voltage.fault: 0.0
input.voltage.nominal: 220
output.voltage: 212.0
ups.alarm: UPS selftest failed!  *************
ups.beeper.status: disabled
ups.delay.shutdown: 30
ups.delay.start: 180
ups.firmware:  Ver 02.9B
ups.load: 78
ups.mfr: UPS
ups.model: Model 3000
ups.status: ALARM OL    **********************
ups.temperature: 29.0
ups.type: online
```

The check_ups plugin reports "OK" condition, see this when I run it manually:

```
./check_ups -H somehost -u ups_name -T
UPS OK - Status=Online, Unknown Utility=209,0V Batt=74,0% Load=79,0% Temp=29,0C|voltage=209000mV;;;0 battery=74%;;;0;100 load=79%;;;0;100 temp=29degC;;;0
```

(notice the "unknown" next to "online", when the alarm clears, this disappears)

We had to replace the fan and put the UPS in operation again.
To further test, I put the same UPS in manual bypass, and I also got "OK" status:

`UPS OK - Status=Online, On Bypass Utility=203,0V Batt=75,0% Load=78,0% Temp=31,0C|voltage=203000mV;;;0 battery=75%;;;0;100 load=78%;;;0;100 temp=31degC;;;0`

The plugin detects the bypass, but displays OK status.

Is this the expected behavior?
I would assume the bypass condition would trigger a WARNING, and the ALARM condition would trigger a CRITICAL status.

Am I doing something wrong?

```
./check_ups -V
check_ups v2.2 (monitoring-plugins 2.2)
```

Thanks in advance!
Julian

**Comment by RincewindsHat** [comment]
Hm, I don't know much about UPSs, but I would expect warning too.
Taking a look at it in time.

**Comment by RincewindsHat** [comment]
Should be fixed by #1961
